flowpm/raytracing.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Feb  3 12:29:19 2021
+
+@author: Denise Lanzieri
+"""
+import numpy as np
+import tensorflow as tf
+import tensorflow_addons as tfa
+import flowpm
+import flowpm.constants as constants
+
+
+def generate_matrix():
+  """ Generate the possible rotation matrices
+    """
+  x = np.asarray([1, 0, 0], dtype=int)
+  y = np.asarray([0, 1, 0], dtype=int)
+  z = np.asarray([0, 0, 1], dtype=int)
+  M_matrices = [np.asarray([x,y,z],dtype=int), np.asarray([x,z,y],dtype=int),np.asarray([z,y,x],dtype=int),np.asarray([z,x,y],dtype=int), \
```

Comment:
the matrices will be used to apply rotations on float tensors, so we probably want to use a dtype of float32 instead of int

Reply:
done